File: client/src/test/java/com/github/dkurata38/open_web_library/client/ImageCognitionClientTest.java
package com.github.dkurata38.open_web_library.client;

import com.github.dkurata38.open_web_library.application.book.ImageCognitionClient;
import lombok.extern.slf4j.Slf4j;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.core.io.Resource;
import org.springframework.core.io.ResourceLoader;

@SpringBootTest
@Slf4j
class ImageCognitionClientTest {

	@Autowired
	private ImageCognitionClient imageCognitionClient;
	@Autowired
	private ResourceLoader resourceLoader;

//	@Test
	void getContents() {
		Resource resource =
				resourceLoader.getResource("classpath:Test2.jpg");
		String contents = imageCognitionClient.extractTextFromImage(resource);
		log.info(contents);
	}
}

File: client/src/test/kotlin/com/github/dkurata38/open_web_library/client/ImageCognitionClientTest.kt
package com.github.dkurata38.open_web_library.client;

import com.github.dkurata38.open_web_library.client.image_congnition.ImageCognitionClient
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.core.io.ResourceLoader

@SpringBootTest
class ImageCognitionClientTest(@Autowired private val imageCognitionClient: ImageCognitionClient,
                               @Autowired private val resourceLoader: ResourceLoader) {

    @Test
    fun getContents() {
        val resource =
                resourceLoader.getResource("classpath:Test2.jpg")
        val contents = imageCognitionClient.extractTextFromImage(resource)
        Assertions.assertNotNull(contents)
    }
}
